code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -100,7 +100,10 @@
 /mob/living/silicon/robot/get_cell()
 	return cell
 
-/mob/living/silicon/robot/Initialize(mapload)
+/mob/living/silicon/robot/New(loc, connect_to_ai = TRUE)
+	. = ..()
```

Comment:
Why are you doing this in new, and what? Why?

Reply:
Can I give new an argument in `Initialize` without putting it in `New`?

Reply:
yes